ThePensionsRegulator.Frontend.Umbraco/App_Plugins/ThePensionsRegulator.Frontend.Umbraco/blocks/views/TPRDocuments.html

```diff
@@ -45,12 +45,34 @@
             <dd ng-if="doc.datePublished != ''">Published: {{doc.datePublished}}</dd>
             <dd ng-bind="doc.description"></dd>
         </div>
+        <div ng-if="doc.document[0].url.endsWith('.rtf')">
+            <dt>
+                <a class="govuk-link" href="{{doc.document[0].url}}">
+                    {{doc.document[0].name}}
+                    <br>
+                    <span class="misc fileicon">RTF</span> --KB, {{doc.numberOfPages}} page(s)
+                </a>
+            </dt>
+            <dd ng-if="doc.datePublished != ''">Published: {{doc.datePublished}}</dd>
+            <dd ng-bind="doc.description"></dd>
+        </div>
+        <div ng-if="doc.document[0].url.endsWith('.odt')">
+            <dt>
+                <a class="govuk-link" href="{{doc.document[0].url}}">
+                    {{doc.document[0].name}}
+                    <br>
+                    <span class="misc fileicon">ODT</span> --KB, {{doc.numberOfPages}} page(s)
+                </a>
+            </dt>
+            <dd ng-if="doc.datePublished != ''">Published: {{doc.datePublished}}</dd>
+            <dd ng-bind="doc.description"></dd>
+        </div>
         <div ng-if="doc.document[0].url.endsWith('.xlsx')">
             <dt>
                 <a class="govuk-link" href="{{doc.document[0].url}}">
                     {{doc.document[0].name}}
                     <br>
-                    <span class="excel fileicon">XLSX</span> --KB
+                    <span class="excel fileicon">EXCEL</span> --KB
```

Comment:
Where something should be announced as a word "Excel" rather than initials "R. T. F." then it should be in sentence case, otherwise a screen-reader will read is as initials. It will still display in uppercase using your CSS `text-transform: uppercase`.